src/components/common/SessionApprove.vue

```diff
@@ -111,11 +97,21 @@ export default {
     tx() {
       return this.signRequest ? parseTx(this.signRequest.signMessage) : null
     },
-    // fees() {
-    //   return this.tx ? this.tx.tx.value.fee.amount : null
-    // },
+    fees() {
+      return this.signRequest ? parseFee(this.signRequest.signMessage) : null
```

Comment:
You may even be able to do away with the `parsers.js` file if you parse it once and then have each of these computed properties just access the object as normal.